velox/functions/sparksql/tests/JsonObjectKeysTest.cpp

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/functions/sparksql/tests/SparkFunctionBaseTest.h"
+
+using namespace facebook::velox::test;
+
+namespace facebook::velox::functions::sparksql::test {
+namespace {
+
+class JsonObjectKeysTest : public SparkFunctionBaseTest {
+ protected:
+  VectorPtr jsonObjectKeys(const std::string& json) {
+    auto varcharVector =
+        makeFlatVector<StringView>({StringView(json)}, VARCHAR());
```

Comment:
nit: StringView(json) -> json
We could use json directly to create a StringView vector.

Reply:
Done, thanks